### DISCUSSION TITLE
upgrade iso to include bug 0015522 centOS

### DISCUSSION THREAD

**Discussion by ingAntoN-71** [Security-Onion-Solutions/securityonion#9133] [General] [answered]
Someone can upgrade iso  to include bug 0015522
![Capture222](https://user-images.githubusercontent.com/118121467/201530838-f6cad351-fda9-467f-94c3-ad6d4aea9a9b.PNG)

**Comment by ingAntoN-71** [comment]
https://bugs.centos.org/view.php?id=15522

**Comment by ingAntoN-71** [comment]
this bug is still open into 2.3.182 version.

**Comment by ingAntoN-71** [comment]
this bug blocks at all a clean install of the 2.3.182 into old servers.

**Comment by dougburks** [comment, marked as answer]
If our Security Onion ISO image doesn't work on your hardware, then please choose one of the alternative installation methods:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
